app/handlers/base.py

```diff
@@ -2,6 +2,7 @@
 import tornado.ioloop
 from sqlalchemy.orm import joinedload
 from sqlalchemy.orm.exc import NoResultFound
+import json.decoder
```

Comment:
Slightly prefer `from json.decoder import JSONDecodeError` just to clarify that we're not calling their decoder directly.